src/main/java/appeng/me/cluster/implementations/CraftingCPUCluster.java

```diff
@@ -153,7 +153,11 @@ void addBlockEntity(CraftingBlockEntity te) {
         } else if (te.isStatus()) {
```

Comment:
Should probably just check for `instanceof CraftingMonitorBlockEntity`, I don't think `isStatus` should be in the interface.